graph.go

```diff
@@ -106,7 +107,7 @@ func (c *GraphCmd) Execute(args []string) error {
 				}
 
 				oldSrcDir := filepath.Join(dir, "src")
-				newGOPATH := filepath.Join("/tmp/gopath-" + strconv.Itoa(i) + "-" + filepath.Base(dir))
+				newGOPATH := path.Join(os.TempDir(), "gopath-"+strconv.Itoa(i)+"-"+filepath.Base(dir))
```

Comment:
Why did you change it from `filepath.Join` to `path.Join`? I think it should be the former.

`path.Join` always uses forward-slashes as separators, whereas `filepath.Join` uses the OS's separator (which may be forward slash, backward slash or potentially something else).